reconcile/sql_query.py

```diff
@@ -208,45 +217,64 @@ def collect_queries(query_name=None):
     return queries_list
 
 
-def make_postgres_command(output, query):
-    command = [
-        'time psql',
-        'postgres://$(db.user):$(db.password)@'
-        '$(db.host):$(db.port)/$(db.name)',
-        f'--command "{query}"',
-    ]
+def make_postgres_command(output, sqlqueries):
+    command = []
+    for query in sqlqueries:
+        command.extend([
+            '(time psql',
+            'postgres://$(db.user):$(db.password)@'
+            '$(db.host):$(db.port)/$(db.name)',
+            f'--command "{query}")',
+        ])
+
+        if output == 'filesystem':
+            command.extend(filesystem_redir_stdout())
+        else:
+            command.append(';')
 
     if output == 'filesystem':
-        command.extend(filesystem_extra_command())
+        command.extend(filesystem_closing_message())
 
     return ' '.join(command)
 
 
-def make_mysql_command(output, query):
-    command = [
-        'time mysql',
-        '--host=$(db.host)',
-        '--port=$(db.port)',
-        '--database=$(db.name)',
-        '--user=$(db.user)',
-        '--password=$(db.password)',
-        f'--execute="{query}"',
-    ]
+def make_mysql_command(output, sqlqueries):
+    command = []
+    for query in sqlqueries:
+        command.extend([
+            '(time mysql',
+            '--host=$(db.host)',
+            '--port=$(db.port)',
+            '--database=$(db.name)',
+            '--user=$(db.user)',
+            '--password=$(db.password)',
+            f'--execute="{query}")',
+        ])
+
+        if output == 'filesystem':
+            command.extend(filesystem_redir_stdout())
+        else:
+            command.append(';')
 
     if output == 'filesystem':
-        command.extend(filesystem_extra_command())
+        command.extend(filesystem_closing_message())
 
     return ' '.join(command)
 
 
-def filesystem_extra_command():
+def filesystem_redir_stdout():
+    return [
+        ' &>> /tmp/query-result.txt;',
+    ]
+
+
+def filesystem_closing_message():
     return [
-        '> /tmp/query-result.txt;',
         'echo;',
         'echo Get the sql-query results with:;',
         'echo;',
-        'echo  oc rsh --shell=/bin/bash ${HOSTNAME} ',
-        'cat /tmp/query-result.txt;',
+        'echo  oc cp ${HOSTNAME}:/tmp/query-result.txt ',
```

Comment:
I get a warning when running this command and it looks like it fails
`tar: Removing leading '/' from member names`
Could we remove the leading '/'?

Reply:
You tell me :)